apps/avifgainmaputil/tonemap_command.cc

```diff
@@ -52,7 +52,7 @@ avifResult TonemapCommand::Run() {
   avifContentLightLevelInformationBox clli_box = {};
   bool clli_set = false;
   if (arg_clli_str_.value().size() > 0) {
-    std::vector<uint32_t> clli;
+    std::vector<uint16_t> clli;
```

Comment:
At lines 62-63 these are assigned to `clli_box.maxCLL` and `clli_box.maxPALL`, both of which are `uint16_t`.

So declaring the `clli` vector as `uint16_t` fixed the warnings.

If the `uint16_t` type is too small, we can change lines 62-63 to:

```
    if (clli[0] > UINT16_MAX || clli[1] > UINT16_MAX) {
      std::cerr << "Huge clli values > 2^16 - 1\n";
      return AVIF_RESULT_INVALID_ARGUMENT;
    }
    clli_box.maxCLL = static_cast<uint16_t>(clli[0]);
    clli_box.maxPALL = static_cast<uint16_t>(clli[1]);
```

Reply:
i think this the current cast is fine.